framework/src/ble_manager/ble_manager_state.c

```diff
@@ -910,6 +910,12 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 			}
 		}
 
+		if (ctx == NULL) {
```

Comment:
If `ctx` is NULL, is it needed to run from 890 to 911?
If not, it is better to move this NULL checking at line 890.

Reply:
@jeongchanKim 
At line:907, ctx is set after matching BLE context table.
This line means a context is not found in the full table.